### PR TITLE
ODSP Archimedes Capacity Scenario Documentation

### PR DESCRIPTION
**Pull request by lohithmadhala**
Project Archimedes Capacity Scenario Documentation Update.

### CODE REVIEW

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+Capacity scenario is a storage use case powered by our available Sharepoint datasets. This allows customers to better understand how their storage is being used by providing meaningful insights and analytics offered by the SharePoint Sites and SharePoint Files datasets
+
+**IMPORTANT NOTE:** This scenario is not currently available publicly. It is coming soon. The details provided here are for informational purposes only.
+
+**After you follow these steps, you will have a great set of Power BI dashboards related to SharePoint Capacity Scenario, like the one shown below.**
+![](Images/Image_1.png) 
+
+## Installing Pre-reqs
+The first step to running this template would be to create an application in the tenant and use that appId 
+and secret to setup the other required resources.
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:21]: Should 20+21 be a single line?

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+3. Save the application id (In the screenshot, the one ending in 9826). Navigate to API permissions
+
+![](Images/3.png)
+
+4. Select "Microsoft Graph" from the Add permission flyout
+
+![](Images/4.png)
+
+5. Select "Application permissions -> Applications -> Application.Read.All"
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:39]: Is this permissions required? I remember getting some push back on this one.

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+1.  After the pre-reqs are complete, navigate to the Synapse workspace just created
+
+![](Images/21.png)
+
+2.  Open the Synapse Studio
+
+![](Images/22.png)
+
+3.  Navigate to "Integrate -> Add new resource ->Browse gallery"
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:107]: nit: space before "Browse"

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    - Link to download Download Microsoft Power BI Desktop from Official Microsoft Download Center. [here](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
+2. Download the pre-created Power BI security report that can generate insights from data that is produced using Synapse pipeline in azure storage locations. Link to download Power BI Report. [here]()
+
+![](Images/42.png)
+
+3. Open the Power BI file and click on Transform data → Data source settings
+
+![](Images/43.png)
+
+4. You will see 1 data sources in the Data source settings page. Change the Storage account path in URL with right storage account that data is generated from synapse pipeline in the steps above. You can get the storage account that is used in Synapse template pipeline Step 6 above
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:181]: Is it 1 data source or 2 data sources?

File: ARMTemplates/Archimedes Capacity Scenario/readme.md (hunk 2 above)
Comment (lohithmadhala) [reply ARMTemplates/Archimedes Capacity Scenario/readme.md:39]: these steps are the same as from the security template. if the instructions have changed there, i can change it here too

File: ARMTemplates/Archimedes Capacity Scenario/readme.md (hunk 4 above)
Comment (lohithmadhala) [reply ARMTemplates/Archimedes Capacity Scenario/readme.md:181]: just 1 - this is the parent directory for both the extracted datasets. inside the pbi template, we split them into 2 queries

**Review by udaykumar54 — APPROVED** [review]

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+12. Trigger the pipeline
+
+![](Images/27c.PNG)
+
+13. Provide the required parameters - StartTime, EndTime, StorageAccount, and StorageContainer created by the pre-req steps above (Note: names are case sensitive)
+
+![](Images/27d.PNG)
+
+14. You ran the pipeline created the request for the SharePoint Sites and SharePoint Files datasets which will be used to power the report! Once the admin consents to the request the data will be processed and delivered to your storage account.
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:160]: Replace "Once the admin consents to the request the data will..."
With "The data will..."

With the new consent model, the admin consents ahead of time, so this is no longer accurate.

File: ARMTemplates/Archimedes Capacity Scenario/readme.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+Note: In order to draw data to the Power BI report, you first need to run the Synapse pipeline as instructed in the previous section.
+
+The report will always pull the latest data from your Azure Data Lake Storage Gen 2. The report is to been as a template and is to be modified based on your requirements 
+
+Below steps will help to link datasets that are generated using Synapse pipeline above to link to Power BI 
+Template. 
+1. Download and install Microsoft Power BI Desktop if you don’t have it installed already on your machine. 
+    - Link to download Download Microsoft Power BI Desktop from Official Microsoft Download Center. [here](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
+2. Download the pre-created Power BI security report that can generate insights from data that is produced using Synapse pipeline in azure storage locations. Link to download Power BI Report. [here]()
```
Comment (Jose-Barreto) [inline ARMTemplates/Archimedes Capacity Scenario/readme.md:176]: Replace "Link to download Power BI Report. [here]()" 
With "The link to download the Power BI Report will be available here when the template becomes available."

File: solutions/capacity-scenario/readme.md [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+
+10. Review the changes and click Publish
+
+![](Images/27a.PNG)
+
+11. Verify that the pipeline has been successfully published
+
+![](Images/27b.PNG)
+
+12. Trigger the pipeline
```
Comment (Jose-Barreto) [inline solutions/capacity-scenario/readme.md:152]: I assume we could add a note here: "Note: You should not run the pipeline until the Files dataset becomes available."

File: solutions/capacity-scenario/readme.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+![](Images/22.png)
+
+1.  Navigate to "Integrate -> Add new resource -> **Browse gallery**"
+
+![](Images/23.png)
+
+4.  Search for "SharePoint" and select the "Unlock Capacity Analytics and Insights for SharePoint and OneDrive using Microsoft 365 Datasets" template and Continue
+
+![](Images/24.png)
+
```
Comment (Jose-Barreto) [inline solutions/capacity-scenario/readme.md:117]: I assume the template will not show here yet.
If so, we should have a note: "Note: You won't see the capacity template here until it becomes available."

**Review by Jose-Barreto — APPROVED** [review]

**Review by udaykumar54 — APPROVED** [review]

**Pull request merged**